resources/js/Components/NavBar/NavBar.vue

```diff
@@ -47,15 +51,15 @@
             </a>
         </div>
         <div class="navbar-center">
-            <img class="block lg:hidden h-8 w-auto"  :src="'images/logo-mini-blue.svg'" /> 
+            <img class="block lg:hidden h-8 w-auto"  :src="'images/logo-mini-blue.svg'" />
         </div>
         <div class="navbar-end">
             <a class="btn btn-ghost btn-circle" href="https://github.com/devpt-org">
                 <GitHubIcon width="20" height="20" />
             </a>
-            <a class="btn btn-ghost btn-circle" @click="toggleDark">
-                <SunIcon v-show="this.dark" width="20" height="20" class="svg-inline--fa fa-moon fa-w-16" />
-                <MoonIcon v-show="!this.dark" width="20" height="20" class="svg-inline--fa fa-moon fa-w-16" />
+            <a class="btn btn-ghost btn-circle" @click="toggleDark()">
+                <SunIcon v-show="!this.dark" width="20" height="20" />
+                <MoonIcon v-show="this.dark" width="20" height="20" />
```

Comment:
Aqui poderíamos usar um `v-else`. Na situação em si far-me-ia sentido não usarmos (isto é, deixarmos como está) pois envolve outras alterações - muda o src da `img` por exemplo).

```suggestion
                <MoonIcon v-show="this.dark" width="20" height="20" />
                <SunIcon v-else width="20" height="20" />
```

O que te parece?